tests/app/phase-team-response/issues-pending/issues-pending.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { IssuesPendingComponent } from '../../../../src/app/phase-team-response/issues-pending/issues-pending.component';
+import { Issue } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+import { IssueService } from '../../../../src/app/core/services/issue.service';
+import { UserService } from '../../../../src/app/core/services/user.service';
+import { USER_Q } from '../../../constants/data.constants';
+
+fdescribe('IssuesPendingComponent', () => {
```

Comment:
Remember to switch back to `describe` before we merge

Reply:
Right, sorry for missing that!😅